test/views/SubmitFormViewHiddenTextSpec.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import java.time.LocalDate
+import forms.SubmitVatReturnForm
+import models.VatObligation
+import models.nrs.{CY, EN}
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.api.i18n.Lang
+
+class SubmitFormViewHiddenTextSpec extends ViewBaseSpec {
+
+  "When the submit form view is rendered " when {
```

Comment:
These tests should just be within the existing SubmitFormViewSpec. You could add the english test to an existing scenario (as they're all english by default), then add the Welsh one as a supplementary scenario.